conf/core/ConfigOptions.asciidoc

```diff
@@ -1496,6 +1496,21 @@ The default alpha value of 1.15 was determined through experimentation with a Ja
 using MeanWordSetDistance as the container classes. See Redmine ticket #2349 for some experiment
 details.  The "best" value varies depending on the input data as well as how the data is being used.
 
+=== libaddressinput.address.rules.file
+
+* Data Type: string
+* Default Value: `${HOOT_HOME}/conf/core/poi-polygon/countryinfo.txt`
+
+JSON rules file used by libaddressinput for address normalization/validation.
+
+=== libpostal.data.dir
+
+* Data Type: string
+* Default Value: `/home/vagrant/libpostal-data/libpostal`
+
+Absolute path to the directory where the libpostal library, used for POI/Polygon address parsing,
+stores its data.
+
```

Comment:
This default dir may need to change based on how devops sets up the install of the rpm.

Reply:
Changed to `/usr/share/libpostal` in a3901b3.